### PR TITLE
fix(block-section): added rtl check and css

### PR DESCRIPTION
**Pull request by asangma**
**Related Issue:** #957

## Summary
Add `calcite--rtl` to BlockSection.
This CSS class on Block wasn't affecting BlockSection.

### CODE REVIEW

**Comment by asangma** [comment]
cc @AdelheidF

File: src/demos/block/basic.html [hunk 1]
```diff
@@ -90,6 +90,18 @@ <h3>Header + content (collapsible)</h3>
               <img alt="demo" src="https://placeimg.com/640/480/nature" />
             </calcite-block-section>
           </calcite-block>
+
+          <h3>With sections</h3>
```
Comment (jcfranco) [inline src/demos/block/basic.html:94]: Should this be  `<h2>Header + content (sections w/ switch)</h2>`?

Comment (asangma) [reply src/demos/block/basic.html:94]: oh yeah.

Comment (asangma) [reply src/demos/block/basic.html]: Updated.

**Pull request merged**